src/components/Button.module.css

```diff
@@ -0,0 +1,20 @@
+button {
```

Comment:
You should use css class here. Using tag as a selector you might make it global, but with css modules we don't want it.